src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -300,8 +300,15 @@ void GcsPlacementGroupManager::OnPlacementGroupCreationSuccess(
   auto placement_group_id = placement_group->GetPlacementGroupID();
   RAY_CHECK_OK(gcs_table_storage_->PlacementGroupTable().Put(
       placement_group_id, placement_group->GetPlacementGroupTableData(),
-      [this, placement_group_id](Status status) {
+      [this, placement_group_id, placement_group](Status status) {
         RAY_CHECK_OK(status);
+        // Update the resource in gcs resource manager
+        for (auto &bundle : placement_group->GetBundles()) {
```

Comment:
Can you add cpp unit tests to verify this? (resources are updated after pg is created)